### ISSUE TITLE
JSON headers being truncated in span

### ISSUE THREAD

**Issue by d-wire** [DataDog/dd-trace-java#6148]
Hey folks, we're trying to add request headers to our traces. To do that, we're adding environment variables like this
```
      - name: DD_TRACE_REQUEST_HEADER_TAGS
        value: <key>:<value>,<key>:<value>
```
However, one of our headers is JSON that looks like this
```
{"user_agent_ip":<value>,"user_agent":<value>}
```
The span tags don't seem to like JSON headers, and instead of displaying them as normal it seems to be truncating the value at the comma, so that it looks like this in the trace
```
{"user_agent_ip":"<value>"
```
Is there a way to get the trace to not truncate these JSON headers? Thanks in advance!

**Comment by bantonsson** [comment]
Hi @d-wire. We have decided to switch back to being lenient with these tag values containing commas. The fix is in progress.

**Comment by bantonsson** [comment]
This has now been merged and the fix will be part of the `1.26.0` release.
